migtests/tests/pg/assessment-report-test/init-db

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+source ${SCRIPTS}/functions.sh
+
+
+echo "Creating ${SOURCE_DB_NAME} database on source"
+run_psql postgres "DROP DATABASE IF EXISTS ${SOURCE_DB_NAME};"
+run_psql postgres "CREATE DATABASE ${SOURCE_DB_NAME};"
+
+echo "Initialising source database."
+
+run_psql ${SOURCE_DB_NAME} "\i ${TEST_DIR}/../misc-objects-1/schema.sql;"
+run_psql ${SOURCE_DB_NAME} "\i ${TEST_DIR}/../misc-objects-2/pg_misc_objects2.sql;"
+run_psql ${SOURCE_DB_NAME} "\i ${TEST_DIR}/../views-and-rules/pg_views_and_rules_automation.sql;"
+run_psql ${SOURCE_DB_NAME} "\i pg_assessment_report.sql;"
+
+echo "Running on schema2"
+TEMP_SQL=$(mktemp "${TEST_DIR}/temp_sql.XXXXXX")
+
+if [ ! -f "$TEMP_SQL" ]; then
+  echo "Failed to create temporary file in ${TEST_DIR}"
+  exit 1
+fi
+
+# Write the SQL commands to the temporary SQL script
```

Comment:
The below comment might be more useful
nit: `Writing temporary SQL script to create DB objects for the test`